sphinx/source/docs/first_steps/examples/first_steps_4_grid_lines.py

```diff
@@ -15,10 +15,10 @@
 # add a renderer
 p.line(x, y, line_color="green", line_width=2)
 
-# change just some things about the x-grid
+# change line color of the x-grid
 p.xgrid.grid_line_color = "red"
 
-# change just some things about the y-grid
+# change grid line attributes of the y-grid
```

Comment:
cc @tcmetzger FWIW the wording here with "just" was intentional, I was trying to emphasize that the first line *only* affected the x-axis, while the second *only* affected the y-axis.

Reply:
I see, perhaps I could change it by using the word _only_, omitting the _just_